recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -1,10 +1,29 @@
-SRC_URI += "file://weston.ini"
+inherit systemd
+
+SRC_URI += "file://weston.ini \
+            file://weston-start \
+            file://weston.service \
+            file://weston.path \
+            "
 
 do_install_append() {
     install -D -p -m0644 ${WORKDIR}/weston.ini ${D}${sysconfdir}/xdg/weston/weston.ini
+
+    # Install weston-start script
+    install -Dm755 ${WORKDIR}/weston-start ${D}${bindir}/weston-start
+    sed -i 's,@DATADIR@,${datadir},g' ${D}${bindir}/weston-start
```

Comment:
can you use the same separator for sed? my personal preference being pipe ("|") instead of "," or ":".